### PR TITLE
 Bug 1872280: Rename GitOps in nav to Application Stages

### PR DESCRIPTION
**Pull request by divyanshiGupta**
**Jira:** https://issues.redhat.com/browse/ODC-4503

**Screenshot:**
image

### CODE REVIEW

**Comment by divyanshiGupta** [comment]
cc: @serenamarie125 @bkrikori

File: frontend/packages/dev-console/src/plugin.tsx [hunk 1]
```diff
@@ -236,6 +221,21 @@ const plugin: Plugin<ConsumedExtensions> = [
...
+      perspective: 'dev',
+      group: 'resources',
+      componentProps: {
+        name: 'Application Stages',
+        href: '/applicationstages',
+        testID: 'application-stages-header',
+      },
+    },
+    flags: {
+      required: [FLAG_OPENSHIFT_GITOPS],
```
Comment (sahil143) [inline frontend/packages/dev-console/src/plugin.tsx:236]: Why not rename the flag as well 
```suggestion
      required: [FLAG_OPENSHIFT_APPLICATION_STAGES],
```

**Review by sahil143 — CHANGES_REQUESTED** [review]

**Comment by jerolimov** [comment]
In general it lgtm for the "smallest possible change"..

I'm unsure if we want rename also all the GitOps files? I expect that the answer is no, but I do not want make the decision here... @christianvogt? :thinking:

**Comment by christianvogt** [comment]
I think we're fine keeping the code named around 'gitops' as it's much more identifiable than 'application stages'

**Comment by jerolimov** [comment]
If you create and link a bugzilla ticket this will get a lgtm from me. :smile:

File: frontend/packages/dev-console/src/plugin.tsx (hunk 1 above)
Comment (divyanshiGupta) [reply frontend/packages/dev-console/src/plugin.tsx:236]: The operator is still GitOps and the flag can also be used at multiple places and not just for `Application Stages` page.

**Comment by divyanshiGupta** [comment]
cc: @jerolimov

**Comment by jerolimov** [comment]
:+1:

/lgtm

**Pull request merged**